pkgs/applications/misc/easystroke/default.nix

```diff
@@ -0,0 +1,55 @@
+{ stdenv, fetchFromGitHub, fetchpatch, pkgconfig, boost, dbus_glib, gnome3,
+ xorg, help2man, intltool, libsigcxx, gcc7, wrapGAppsHook}: 
+
+with stdenv.lib;
+
+let
+  version = "0.6.0-9-gf7c1614";
+in stdenv.mkDerivation {
+  name = "easystroke-${version}";
+
+  src = fetchFromGitHub {
+    owner = "thjaeger";
+    repo = "easystroke";
+    rev = "f7c1614004e9c518bd4f6f4b3a2ddaf23911a5ef";
+    sha256 = "0map8zbnq993gchgw97blf085cbslry2sa3z4ambdcwbl0r9rd6x";
+  };
+
+  nativeBuildInputs = [ pkgconfig gcc7 wrapGAppsHook help2man ];
+  buildInputs = [ boost dbus_glib gnome3.gtkmm xorg.libXtst xorg.inputproto 
```

Comment:
Can you fix the indention & order here?

Sorting dependencies alphabetically and having a common indention level across all list elements is what most people seem to do. Unfortunately, there is no style guide (yet).